src/test/java/spoon/test/reference/ElasticsearchStackoverflowTest.java

```diff
@@ -46,7 +46,6 @@ public <T> void visitCtExecutableReference(
 		}
 	}
 
-	@GitHubIssue(issueNumber = 5977, fixed = false)
```

Comment:
Why do you delete this outright? Is it fixed? If so, why not set `fixed = true`?

Reply:
I wasn't sure about that one, but the test itself isn't really related to the linked PR, it was just broken temporarily.

Reply:
Ah, I remember now. That was my idea in the first place (and seems to have worked nicely) :^)